msg-app/src/components/Search.jsx

```diff
@@ -1,30 +1,58 @@
+// search broke when parsing data, will be implemented again at a later stage
+// uses old Data.json file
 import { useState } from 'react'
+import search from './assets/search.svg';
+import close from './assets/close.svg'
 
 const Search = ({ placeholder, data }) => {
   const [filtredData, setfiltredData] = useState([]);
+  const [wordEntered, setWordEntered] = useState('');
 
```

Comment:
It seems to be a minor typo here.
Did you mean `filteredData` and `setFilteredData`?